### PR TITLE
go/bundle: add README.md

### PR DESCRIPTION
**Pull request by nyaxt**

### CODE REVIEW

File: go/bundle/README.md [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+## Getting Started
+
+### Prerequisite
+golang environment needs to be set up in prior to using the tool. We are testing the tool on latest golang. Please refer to [Go Getting Started documentation](https://golang.org/doc/install) for the details.
+
+### Installation
+We recommend using `go get` to install the command-line tool.
+
+```
+go get -u github.com/WICG/webpackage/go/bundle/cmd/{gen,dump}-bundle
```
Comment (hajimehoshi) [inline go/bundle/README.md]: `go get -u github.com/WICG/webpackage/go/bundle/cmd/...` would work to install all the binaries.

File: go/bundle/README.md [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+
+### Usage
+`gen-bundle` generates a bundled exchange file from a HAR file.
+
+One convenient way to generate HAR file is via Chrome Devtools. Navigate to "Network" panel, and right-click on any resource and select "Save as HAR with content".
+![generating har with devtools](https://raw.githubusercontent.com/WICG/webpackage/master/go/bundle/har-devtools.png)
+
+Once you have the har file, generate the bundled exchange file via:
+```
+gen-bundle -i foo.har -o foo.webbundle
```
Comment (kinu) [inline go/bundle/README.md]: Should `foo.webbundle` be `foo.wbn` per https://wicg.github.io/webpackage/draft-yasskin-wpack-bundled-exchanges.html#rfc.section.6.1 ?

File: go/bundle/README.md (hunk 1 above)
Comment (nyaxt) [reply go/bundle/README.md]: Done.

**Comment by kinu** [comment]
lgtm, thanks!

**Pull request merged**